src/prof.c

```diff
@@ -1705,11 +1710,76 @@ prof_getpid(void) {
 static bool
 prof_dump_maps(bool propagate_err) {
 	bool ret;
-	int mfd;
+	int mfd = -1;
 
 	cassert(config_prof);
 #ifdef __FreeBSD__
-	mfd = prof_open_maps("/proc/curproc/map");
+	int mib[4];
+	size_t sz;
+	char *buf;
+
+	mib[0] = CTL_KERN;
+	mib[1] = KERN_PROC;
+	mib[2] = KERN_PROC_VMMAP;
+	mib[3] = prof_getpid();
+
+	if (prof_dump_write(propagate_err, "\nMAPPED_LIBRARIES:\n") &&
+	    propagate_err) {
+		ret = true;
+		goto label_return;
+	}
+
+	if (sysctl(mib, 4, NULL, &sz, NULL, 0) == -1) {
+		ret = true;
```

Comment:
Just checked again, it's actually failing on this path, and apparently non-deterministic (~half the time the `sysctl` fails and returns -1). @devnexen any insights?

Here's some context on our FreeBSD CI host:

(gdb) info locals 
ret = true
mfd = -1
mib = 0x7fffdfbfb380
sz = 8234
buf = 0x8007ab000 ""
low = 0x7fffdfbfb4d0 "@]z"
high = 0x8007a5d40 "\020??\001\b"
(gdb) p mib[0]
$1 = 1
(gdb) p mib[1]
$2 = 14
(gdb) p mib[2]
$3 = 32
(gdb) p mib[3]
$4 = 38217

Reply:
Tested in a completely different machine, this time a real FreeBSD release.

`uname -r
12.0-RELEASE-p4`

and ran gmake check couplf of times.


`test_a0 (non-reentrant): pass
--- pass: 1/1, skip: 0/1, fail: 0/1 ---
test_arena_reset (non-reentrant): pass
test_arena_reset (libc-reentrant): pass
test_arena_reset (arena_new-reentrant): pass
test_arena_destroy_initial (non-reentrant): pass
test_arena_destroy_initial (libc-reentrant): pass
test_arena_destroy_initial (arena_new-reentrant): pass
test_arena_destroy_hooks_default (non-reentrant): pass
test_arena_destroy_hooks_default (libc-reentrant): pass
test_arena_destroy_hooks_default (arena_new-reentrant): pass
test_arena_destroy_hooks_unmap (non-reentrant): pass
test_arena_destroy_hooks_unmap (libc-reentrant): pass
test_arena_destroy_hooks_unmap (arena_new-reentrant): pass
--- pass: 12/12, skip: 0/12, fail: 0/12 ---
test_atomic_u64 (non-reentrant): pass
test_atomic_u64 (libc-reentrant): pass
test_atomic_u64 (arena_new-reentrant): pass
test_atomic_u32 (non-reentrant): pass
test_atomic_u32 (libc-reentrant): pass
test_atomic_u32 (arena_new-reentrant): pass
test_atomic_p (non-reentrant): pass
test_atomic_p (libc-reentrant): pass
test_atomic_p (arena_new-reentrant): pass
test_atomic_zu (non-reentrant): pass
test_atomic_zu (libc-reentrant): pass
test_atomic_zu (arena_new-reentrant): pass
test_atomic_zd (non-reentrant): pass
test_atomic_zd (libc-reentrant): pass
test_atomic_zd (arena_new-reentrant): pass
test_atomic_u (non-reentrant): pass
test_atomic_u (libc-reentrant): pass
test_atomic_u (arena_new-reentrant): pass
--- pass: 18/18, skip: 0/18, fail: 0/18 ---
test_background_thread_ctl (non-reentrant): pass
test_background_thread_running (non-reentrant): pass
--- pass: 2/2, skip: 0/2, fail: 0/2 ---
test_deferred (non-reentrant): pass
test_max_background_threads (non-reentrant): pass
--- pass: 2/2, skip: 0/2, fail: 0/2 ---
test_base_hooks_default (non-reentrant): pass
test_base_hooks_default (libc-reentrant): pass
test_base_hooks_default (arena_new-reentrant): pass
test_base_hooks_null (non-reentrant): pass
test_base_hooks_null (libc-reentrant): pass
test_base_hooks_null (arena_new-reentrant): pass
test_base_hooks_not_null (non-reentrant): pass
test_base_hooks_not_null (libc-reentrant): pass
test_base_hooks_not_null (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
test_bitmap_initializer (non-reentrant): pass
test_bitmap_initializer (libc-reentrant): pass
test_bitmap_initializer (arena_new-reentrant): pass
test_bitmap_size (non-reentrant): pass
test_bitmap_size (libc-reentrant): pass
test_bitmap_size (arena_new-reentrant): pass
test_bitmap_init (non-reentrant): pass
test_bitmap_init (libc-reentrant): pass
test_bitmap_init (arena_new-reentrant): pass
test_bitmap_set (non-reentrant): pass
test_bitmap_set (libc-reentrant): pass
test_bitmap_set (arena_new-reentrant): pass
test_bitmap_unset (non-reentrant): pass
test_bitmap_unset (libc-reentrant): pass
test_bitmap_unset (arena_new-reentrant): pass
test_bitmap_xfu (non-reentrant): pass
test_bitmap_xfu (libc-reentrant): pass
test_bitmap_xfu (arena_new-reentrant): pass
--- pass: 18/18, skip: 0/18, fail: 0/18 ---
test_pow2_ceil_u64 (non-reentrant): pass
test_pow2_ceil_u64 (libc-reentrant): pass
test_pow2_ceil_u64 (arena_new-reentrant): pass
test_pow2_ceil_u32 (non-reentrant): pass
test_pow2_ceil_u32 (libc-reentrant): pass
test_pow2_ceil_u32 (arena_new-reentrant): pass
test_pow2_ceil_zu (non-reentrant): pass
test_pow2_ceil_zu (libc-reentrant): pass
test_pow2_ceil_zu (arena_new-reentrant): pass
test_lg_ceil_floor (non-reentrant): pass
test_lg_ceil_floor (libc-reentrant): pass
test_lg_ceil_floor (arena_new-reentrant): pass
--- pass: 12/12, skip: 0/12, fail: 0/12 ---
<jemalloc>: Invalid conf pair: bin_shards:1-160:16|129-512:4|256-256:8
test_bin_shard (non-reentrant): pass
test_bin_shard_mt (non-reentrant): pass
test_producer_consumer (non-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_new_delete (non-reentrant): pass
test_new_delete (libc-reentrant): pass
test_new_delete (arena_new-reentrant): pass
test_count_insert_search_remove (non-reentrant): pass
test_count_insert_search_remove (libc-reentrant): pass
test_count_insert_search_remove (arena_new-reentrant): pass
test_insert_iter_remove (non-reentrant): pass
test_insert_iter_remove (libc-reentrant): pass
test_insert_iter_remove (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
test_decay_ticks (non-reentrant): pass
test_decay_ticks (libc-reentrant): pass
test_decay_ticks (arena_new-reentrant): pass
test_decay_ticker (non-reentrant): pass
test_decay_ticker (libc-reentrant): pass
test_decay_ticker (arena_new-reentrant): pass
test_decay_nonmonotonic (non-reentrant): pass
test_decay_nonmonotonic (libc-reentrant): pass
test_decay_nonmonotonic (arena_new-reentrant): pass
test_decay_now (non-reentrant): pass
test_decay_now (libc-reentrant): pass
test_decay_now (arena_new-reentrant): pass
test_decay_never (non-reentrant): pass
test_decay_never (libc-reentrant): pass
test_decay_never (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_div_exhaustive (non-reentrant): pass
--- pass: 1/1, skip: 0/1, fail: 0/1 ---
test_dict (non-reentrant): pass
test_table_printf (non-reentrant): pass
test_nested_dict (non-reentrant): pass
test_types (non-reentrant): pass
test_modal (non-reentrant): pass
test_json_arr (non-reentrant): pass
test_json_nested_arr (non-reentrant): pass
test_table_row (non-reentrant): pass
--- pass: 8/8, skip: 0/8, fail: 0/8 ---
test_small_extent_size (non-reentrant): pass
test_small_extent_size (libc-reentrant): pass
test_small_extent_size (arena_new-reentrant): pass
test_large_extent_size (non-reentrant): pass
test_large_extent_size (libc-reentrant): pass
test_large_extent_size (arena_new-reentrant): pass
test_monotonic (non-reentrant): pass
test_monotonic (libc-reentrant): pass
test_monotonic (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
test_query (non-reentrant): pass
test_query (libc-reentrant): pass
test_query (arena_new-reentrant): pass
test_batch (non-reentrant): pass
test_batch (libc-reentrant): pass
test_batch (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_fork (non-reentrant): pass
test_fork_multithreaded (non-reentrant): pass
--- pass: 2/2, skip: 0/2, fail: 0/2 ---
test_hash_x86_32 (non-reentrant): pass
test_hash_x86_32 (libc-reentrant): pass
test_hash_x86_32 (arena_new-reentrant): pass
test_hash_x86_128 (non-reentrant): pass
test_hash_x86_128 (libc-reentrant): pass
test_hash_x86_128 (arena_new-reentrant): pass
test_hash_x64_128 (non-reentrant): pass
test_hash_x64_128 (libc-reentrant): pass
test_hash_x64_128 (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
test_hooks_basic (non-reentrant): pass
test_hooks_null (non-reentrant): pass
test_hooks_remove (non-reentrant): pass
test_hooks_alloc_simple (non-reentrant): pass
test_hooks_dalloc_simple (non-reentrant): pass
test_hooks_expand_simple (non-reentrant): pass
test_hooks_realloc_as_malloc_or_free (non-reentrant): pass
test_hooks_realloc (non-reentrant): pass
test_hooks_rallocx (non-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
huge_allocation (non-reentrant): pass
huge_allocation (libc-reentrant): pass
huge_allocation (arena_new-reentrant): pass
huge_mallocx (non-reentrant): pass
huge_mallocx (libc-reentrant): pass
huge_mallocx (arena_new-reentrant): pass
huge_bind_thread (non-reentrant): pass
huge_bind_thread (libc-reentrant): pass
huge_bind_thread (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
test_junk_small (non-reentrant): pass
test_junk_small (libc-reentrant): pass
test_junk_small (arena_new-reentrant): pass
test_junk_large (non-reentrant): pass
test_junk_large (libc-reentrant): pass
test_junk_large (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_junk_small (non-reentrant): pass
test_junk_small (libc-reentrant): pass
test_junk_small (arena_new-reentrant): pass
test_junk_large (non-reentrant): pass
test_junk_large (libc-reentrant): pass
test_junk_large (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_junk_small (non-reentrant): pass
test_junk_small (libc-reentrant): pass
test_junk_small (arena_new-reentrant): pass
test_junk_large (non-reentrant): pass
test_junk_large (libc-reentrant): pass
test_junk_large (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_log_disabled:test/unit/log.c:32: Test skipped: (!config_log)
test_log_disabled (non-reentrant): skip
test_log_disabled:test/unit/log.c:32: Test skipped: (!config_log)
test_log_disabled (libc-reentrant): skip
test_log_disabled:test/unit/log.c:32: Test skipped: (!config_log)
test_log_disabled (arena_new-reentrant): skip
test_log_enabled_direct:test/unit/log.c:44: Test skipped: (!config_log)
test_log_enabled_direct (non-reentrant): skip
test_log_enabled_direct:test/unit/log.c:44: Test skipped: (!config_log)
test_log_enabled_direct (libc-reentrant): skip
test_log_enabled_direct:test/unit/log.c:44: Test skipped: (!config_log)
test_log_enabled_direct (arena_new-reentrant): skip
test_log_enabled_indirect:test/unit/log.c:86: Test skipped: (!config_log)
test_log_enabled_indirect (non-reentrant): skip
test_log_enabled_indirect:test/unit/log.c:86: Test skipped: (!config_log)
test_log_enabled_indirect (libc-reentrant): skip
test_log_enabled_indirect:test/unit/log.c:86: Test skipped: (!config_log)
test_log_enabled_indirect (arena_new-reentrant): skip
test_log_enabled_global:test/unit/log.c:136: Test skipped: (!config_log)
test_log_enabled_global (non-reentrant): skip
test_log_enabled_global:test/unit/log.c:136: Test skipped: (!config_log)
test_log_enabled_global (libc-reentrant): skip
test_log_enabled_global:test/unit/log.c:136: Test skipped: (!config_log)
test_log_enabled_global (arena_new-reentrant): skip
test_logs_if_no_init:test/unit/log.c:158: Test skipped: (!config_log)
test_logs_if_no_init (non-reentrant): skip
test_logs_if_no_init:test/unit/log.c:158: Test skipped: (!config_log)
test_logs_if_no_init (libc-reentrant): skip
test_logs_if_no_init:test/unit/log.c:158: Test skipped: (!config_log)
test_logs_if_no_init (arena_new-reentrant): skip
test_log_only_format_string (non-reentrant): pass
test_log_only_format_string (libc-reentrant): pass
test_log_only_format_string (arena_new-reentrant): pass
--- pass: 3/18, skip: 15/18, fail: 0/18 ---
test_mallctl_errors (non-reentrant): pass
test_mallctl_errors (libc-reentrant): pass
test_mallctl_errors (arena_new-reentrant): pass
test_mallctlnametomib_errors (non-reentrant): pass
test_mallctlnametomib_errors (libc-reentrant): pass
test_mallctlnametomib_errors (arena_new-reentrant): pass
test_mallctlbymib_errors (non-reentrant): pass
test_mallctlbymib_errors (libc-reentrant): pass
test_mallctlbymib_errors (arena_new-reentrant): pass
test_mallctl_read_write (non-reentrant): pass
test_mallctl_read_write (libc-reentrant): pass
test_mallctl_read_write (arena_new-reentrant): pass
test_mallctlnametomib_short_mib (non-reentrant): pass
test_mallctlnametomib_short_mib (libc-reentrant): pass
test_mallctlnametomib_short_mib (arena_new-reentrant): pass
test_mallctl_config (non-reentrant): pass
test_mallctl_config (libc-reentrant): pass
test_mallctl_config (arena_new-reentrant): pass
test_mallctl_opt (non-reentrant): pass
test_mallctl_opt (libc-reentrant): pass
test_mallctl_opt (arena_new-reentrant): pass
test_manpage_example (non-reentrant): pass
test_manpage_example (libc-reentrant): pass
test_manpage_example (arena_new-reentrant): pass
test_tcache_none (non-reentrant): pass
test_tcache_none (libc-reentrant): pass
test_tcache_none (arena_new-reentrant): pass
test_tcache (non-reentrant): pass
test_tcache (libc-reentrant): pass
test_tcache (arena_new-reentrant): pass
test_thread_arena (non-reentrant): pass
test_thread_arena (libc-reentrant): pass
test_thread_arena (arena_new-reentrant): pass
test_arena_i_initialized (non-reentrant): pass
test_arena_i_initialized (libc-reentrant): pass
test_arena_i_initialized (arena_new-reentrant): pass
test_arena_i_dirty_decay_ms (non-reentrant): pass
test_arena_i_dirty_decay_ms (libc-reentrant): pass
test_arena_i_dirty_decay_ms (arena_new-reentrant): pass
test_arena_i_muzzy_decay_ms (non-reentrant): pass
test_arena_i_muzzy_decay_ms (libc-reentrant): pass
test_arena_i_muzzy_decay_ms (arena_new-reentrant): pass
test_arena_i_purge (non-reentrant): pass
test_arena_i_purge (libc-reentrant): pass
test_arena_i_purge (arena_new-reentrant): pass
test_arena_i_decay (non-reentrant): pass
test_arena_i_decay (libc-reentrant): pass
test_arena_i_decay (arena_new-reentrant): pass
test_arena_i_dss (non-reentrant): pass
test_arena_i_dss (libc-reentrant): pass
test_arena_i_dss (arena_new-reentrant): pass
test_arena_i_retain_grow_limit:test/unit/mallctl.c:576: Test skipped: (!retain_enabled)
test_arena_i_retain_grow_limit (non-reentrant): skip
test_arena_i_retain_grow_limit:test/unit/mallctl.c:576: Test skipped: (!retain_enabled)
test_arena_i_retain_grow_limit (libc-reentrant): skip
test_arena_i_retain_grow_limit:test/unit/mallctl.c:576: Test skipped: (!retain_enabled)
test_arena_i_retain_grow_limit (arena_new-reentrant): skip
test_arenas_dirty_decay_ms (non-reentrant): pass
test_arenas_dirty_decay_ms (libc-reentrant): pass
test_arenas_dirty_decay_ms (arena_new-reentrant): pass
test_arenas_muzzy_decay_ms (non-reentrant): pass
test_arenas_muzzy_decay_ms (libc-reentrant): pass
test_arenas_muzzy_decay_ms (arena_new-reentrant): pass
test_arenas_constants (non-reentrant): pass
test_arenas_constants (libc-reentrant): pass
test_arenas_constants (arena_new-reentrant): pass
test_arenas_bin_constants (non-reentrant): pass
test_arenas_bin_constants (libc-reentrant): pass
test_arenas_bin_constants (arena_new-reentrant): pass
test_arenas_lextent_constants (non-reentrant): pass
test_arenas_lextent_constants (libc-reentrant): pass
test_arenas_lextent_constants (arena_new-reentrant): pass
test_arenas_create (non-reentrant): pass
test_arenas_create (libc-reentrant): pass
test_arenas_create (arena_new-reentrant): pass
test_arenas_lookup (non-reentrant): pass
test_arenas_lookup (libc-reentrant): pass
test_arenas_lookup (arena_new-reentrant): pass
test_stats_arenas (non-reentrant): pass
test_stats_arenas (libc-reentrant): pass
test_stats_arenas (arena_new-reentrant): pass
test_hooks (non-reentrant): pass
test_hooks (libc-reentrant): pass
test_hooks (arena_new-reentrant): pass
test_hooks_exhaustion (non-reentrant): pass
test_hooks_exhaustion (libc-reentrant): pass
test_hooks_exhaustion (arena_new-reentrant): pass
--- pass: 81/84, skip: 3/84, fail: 0/84 ---
test_malloc_strtoumax_no_endptr (non-reentrant): pass
test_malloc_strtoumax_no_endptr (libc-reentrant): pass
test_malloc_strtoumax_no_endptr (arena_new-reentrant): pass
test_malloc_strtoumax (non-reentrant): pass
test_malloc_strtoumax (libc-reentrant): pass
test_malloc_strtoumax (arena_new-reentrant): pass
test_malloc_snprintf_truncated (non-reentrant): pass
test_malloc_snprintf_truncated (libc-reentrant): pass
test_malloc_snprintf_truncated (arena_new-reentrant): pass
test_malloc_snprintf (non-reentrant): pass
test_malloc_snprintf (libc-reentrant): pass
test_malloc_snprintf (arena_new-reentrant): pass
--- pass: 12/12, skip: 0/12, fail: 0/12 ---
test_ln_gamma_factorial (non-reentrant): pass
test_ln_gamma_factorial (libc-reentrant): pass
test_ln_gamma_factorial (arena_new-reentrant): pass
test_ln_gamma_misc (non-reentrant): pass
test_ln_gamma_misc (libc-reentrant): pass
test_ln_gamma_misc (arena_new-reentrant): pass
test_pt_norm (non-reentrant): pass
test_pt_norm (libc-reentrant): pass
test_pt_norm (arena_new-reentrant): pass
test_pt_chi2 (non-reentrant): pass
test_pt_chi2 (libc-reentrant): pass
test_pt_chi2 (arena_new-reentrant): pass
test_pt_gamma_shape (non-reentrant): pass
test_pt_gamma_shape (libc-reentrant): pass
test_pt_gamma_shape (arena_new-reentrant): pass
test_pt_gamma_scale (non-reentrant): pass
test_pt_gamma_scale (libc-reentrant): pass
test_pt_gamma_scale (arena_new-reentrant): pass
--- pass: 18/18, skip: 0/18, fail: 0/18 ---
test_mq_basic (non-reentrant): pass
test_mq_basic (libc-reentrant): pass
test_mq_basic (arena_new-reentrant): pass
test_mq_threaded (non-reentrant): pass
test_mq_threaded (libc-reentrant): pass
test_mq_threaded (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_mtx_basic (non-reentrant): pass
test_mtx_basic (libc-reentrant): pass
test_mtx_basic (arena_new-reentrant): pass
test_mtx_race (non-reentrant): pass
test_mtx_race (libc-reentrant): pass
test_mtx_race (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_pack (non-reentrant): pass
test_pack (libc-reentrant): pass
test_pack (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_pages_huge (non-reentrant): pass
test_pages_huge (libc-reentrant): pass
test_pages_huge (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_ph_empty (non-reentrant): pass
test_ph_empty (libc-reentrant): pass
test_ph_empty (arena_new-reentrant): pass
test_ph_random (non-reentrant): pass
test_ph_random (libc-reentrant): pass
test_ph_random (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_prng_lg_range_u32_nonatomic (non-reentrant): pass
test_prng_lg_range_u32_nonatomic (libc-reentrant): pass
test_prng_lg_range_u32_nonatomic (arena_new-reentrant): pass
test_prng_lg_range_u32_atomic (non-reentrant): pass
test_prng_lg_range_u32_atomic (libc-reentrant): pass
test_prng_lg_range_u32_atomic (arena_new-reentrant): pass
test_prng_lg_range_u64_nonatomic (non-reentrant): pass
test_prng_lg_range_u64_nonatomic (libc-reentrant): pass
test_prng_lg_range_u64_nonatomic (arena_new-reentrant): pass
test_prng_lg_range_zu_nonatomic (non-reentrant): pass
test_prng_lg_range_zu_nonatomic (libc-reentrant): pass
test_prng_lg_range_zu_nonatomic (arena_new-reentrant): pass
test_prng_lg_range_zu_atomic (non-reentrant): pass
test_prng_lg_range_zu_atomic (libc-reentrant): pass
test_prng_lg_range_zu_atomic (arena_new-reentrant): pass
test_prng_range_u32_nonatomic (non-reentrant): pass
test_prng_range_u32_nonatomic (libc-reentrant): pass
test_prng_range_u32_nonatomic (arena_new-reentrant): pass
test_prng_range_u32_atomic (non-reentrant): pass
test_prng_range_u32_atomic (libc-reentrant): pass
test_prng_range_u32_atomic (arena_new-reentrant): pass
test_prng_range_u64_nonatomic (non-reentrant): pass
test_prng_range_u64_nonatomic (libc-reentrant): pass
test_prng_range_u64_nonatomic (arena_new-reentrant): pass
test_prng_range_zu_nonatomic (non-reentrant): pass
test_prng_range_zu_nonatomic (libc-reentrant): pass
test_prng_range_zu_nonatomic (arena_new-reentrant): pass
test_prng_range_zu_atomic (non-reentrant): pass
test_prng_range_zu_atomic (libc-reentrant): pass
test_prng_range_zu_atomic (arena_new-reentrant): pass
--- pass: 30/30, skip: 0/30, fail: 0/30 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_accum:true
<jemalloc>: Invalid conf pair: prof_active:false
<jemalloc>: Invalid conf pair: lg_prof_sample:0
test_idump (non-reentrant): pass
--- pass: 1/1, skip: 0/1, fail: 0/1 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_thread_active_init:false
<jemalloc>: Invalid conf pair: lg_prof_sample:0
test_prof_active (non-reentrant): pass
--- pass: 1/1, skip: 0/1, fail: 0/1 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
<jemalloc>: Invalid conf pair: prof_gdump:true
test_gdump (non-reentrant): pass
--- pass: 1/1, skip: 0/1, fail: 0/1 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_accum:true
<jemalloc>: Invalid conf pair: prof_active:false
<jemalloc>: Invalid conf pair: lg_prof_sample:0
<jemalloc>: Invalid conf pair: lg_prof_interval:0
test_idump (non-reentrant): pass
test_idump (libc-reentrant): pass
test_idump (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: lg_prof_sample:0
test_prof_log_many_logs (non-reentrant): pass
test_prof_log_many_traces (non-reentrant): pass
test_prof_log_many_threads (non-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
<jemalloc>: Invalid conf pair: lg_prof_sample:0
test_prof_reset_basic (non-reentrant): pass
test_prof_reset_cleanup (non-reentrant): pass
test_prof_reset (non-reentrant): pass
test_xallocx (non-reentrant): pass
--- pass: 4/4, skip: 0/4, fail: 0/4 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: lg_prof_sample:0
test_prof_realloc (non-reentrant): pass
--- pass: 1/1, skip: 0/1, fail: 0/1 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_prof_thread_name_validation (non-reentrant): pass
test_prof_thread_name_validation (libc-reentrant): pass
test_prof_thread_name_validation (arena_new-reentrant): pass
test_prof_thread_name_threaded (non-reentrant): pass
test_prof_thread_name_threaded (libc-reentrant): pass
test_prof_thread_name_threaded (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_ql_empty (non-reentrant): pass
test_ql_empty (libc-reentrant): pass
test_ql_empty (arena_new-reentrant): pass
test_ql_tail_insert (non-reentrant): pass
test_ql_tail_insert (libc-reentrant): pass
test_ql_tail_insert (arena_new-reentrant): pass
test_ql_tail_remove (non-reentrant): pass
test_ql_tail_remove (libc-reentrant): pass
test_ql_tail_remove (arena_new-reentrant): pass
test_ql_head_insert (non-reentrant): pass
test_ql_head_insert (libc-reentrant): pass
test_ql_head_insert (arena_new-reentrant): pass
test_ql_head_remove (non-reentrant): pass
test_ql_head_remove (libc-reentrant): pass
test_ql_head_remove (arena_new-reentrant): pass
test_ql_insert (non-reentrant): pass
test_ql_insert (libc-reentrant): pass
test_ql_insert (arena_new-reentrant): pass
--- pass: 18/18, skip: 0/18, fail: 0/18 ---
test_qr_one (non-reentrant): pass
test_qr_one (libc-reentrant): pass
test_qr_one (arena_new-reentrant): pass
test_qr_after_insert (non-reentrant): pass
test_qr_after_insert (libc-reentrant): pass
test_qr_after_insert (arena_new-reentrant): pass
test_qr_remove (non-reentrant): pass
test_qr_remove (libc-reentrant): pass
test_qr_remove (arena_new-reentrant): pass
test_qr_before_insert (non-reentrant): pass
test_qr_before_insert (libc-reentrant): pass
test_qr_before_insert (arena_new-reentrant): pass
test_qr_meld_split (non-reentrant): pass
test_qr_meld_split (libc-reentrant): pass
test_qr_meld_split (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_rb_empty (non-reentrant): pass
test_rb_empty (libc-reentrant): pass
test_rb_empty (arena_new-reentrant): pass
test_rb_random (non-reentrant): pass
test_rb_random (libc-reentrant): pass
test_rb_random (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_retained (non-reentrant): pass
test_retained (libc-reentrant): pass
test_retained (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_rtree_read_empty (non-reentrant): pass
test_rtree_read_empty (libc-reentrant): pass
test_rtree_read_empty (arena_new-reentrant): pass
test_rtree_extrema (non-reentrant): pass
test_rtree_extrema (libc-reentrant): pass
test_rtree_extrema (arena_new-reentrant): pass
test_rtree_bits (non-reentrant): pass
test_rtree_bits (libc-reentrant): pass
test_rtree_bits (arena_new-reentrant): pass
test_rtree_random (non-reentrant): pass
test_rtree_random (libc-reentrant): pass
test_rtree_random (arena_new-reentrant): pass
--- pass: 12/12, skip: 0/12, fail: 0/12 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: lg_prof_sample:0
test_malloc_free_overflow (non-reentrant): pass
test_malloc_free_overflow (libc-reentrant): pass
test_malloc_free_overflow (arena_new-reentrant): pass
test_mallocx_dallocx_overflow (non-reentrant): pass
test_mallocx_dallocx_overflow (libc-reentrant): pass
test_mallocx_dallocx_overflow (arena_new-reentrant): pass
test_malloc_sdallocx_overflow (non-reentrant): pass
test_malloc_sdallocx_overflow (libc-reentrant): pass
test_malloc_sdallocx_overflow (arena_new-reentrant): pass
test_realloc_overflow (non-reentrant): pass
test_realloc_overflow (libc-reentrant): pass
test_realloc_overflow (arena_new-reentrant): pass
test_rallocx_overflow (non-reentrant): pass
test_rallocx_overflow (libc-reentrant): pass
test_rallocx_overflow (arena_new-reentrant): pass
test_xallocx_overflow (non-reentrant): pass
test_xallocx_overflow (libc-reentrant): pass
test_xallocx_overflow (arena_new-reentrant): pass
test_realloc_no_overflow (non-reentrant): pass
test_realloc_no_overflow (libc-reentrant): pass
test_realloc_no_overflow (arena_new-reentrant): pass
test_rallocx_no_overflow (non-reentrant): pass
test_rallocx_no_overflow (libc-reentrant): pass
test_rallocx_no_overflow (arena_new-reentrant): pass
--- pass: 24/24, skip: 0/24, fail: 0/24 ---
test_seq_simple (non-reentrant): pass
test_seq_threaded (non-reentrant): pass
--- pass: 2/2, skip: 0/2, fail: 0/2 ---
test_gen_rand_32 (non-reentrant): pass
test_gen_rand_32 (libc-reentrant): pass
test_gen_rand_32 (arena_new-reentrant): pass
test_by_array_32 (non-reentrant): pass
test_by_array_32 (libc-reentrant): pass
test_by_array_32 (arena_new-reentrant): pass
test_gen_rand_64 (non-reentrant): pass
test_gen_rand_64 (libc-reentrant): pass
test_gen_rand_64 (arena_new-reentrant): pass
test_by_array_64 (non-reentrant): pass
test_by_array_64 (libc-reentrant): pass
test_by_array_64 (arena_new-reentrant): pass
--- pass: 12/12, skip: 0/12, fail: 0/12 ---
test_update_slab_size (non-reentrant): pass
test_update_slab_size (libc-reentrant): pass
test_update_slab_size (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_size_classes (non-reentrant): pass
test_size_classes (libc-reentrant): pass
test_size_classes (arena_new-reentrant): pass
test_psize_classes (non-reentrant): pass
test_psize_classes (libc-reentrant): pass
test_psize_classes (arena_new-reentrant): pass
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
test_arena_slab_regind (non-reentrant): pass
test_arena_slab_regind (libc-reentrant): pass
test_arena_slab_regind (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_smoothstep_integral (non-reentrant): pass
test_smoothstep_integral (libc-reentrant): pass
test_smoothstep_integral (arena_new-reentrant): pass
test_smoothstep_monotonic (non-reentrant): pass
test_smoothstep_monotonic (libc-reentrant): pass
test_smoothstep_monotonic (arena_new-reentrant): pass
test_smoothstep_slope (non-reentrant): pass
test_smoothstep_slope (libc-reentrant): pass
test_smoothstep_slope (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
test_spin (non-reentrant): pass
test_spin (libc-reentrant): pass
test_spin (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_stats_summary (non-reentrant): pass
test_stats_large (non-reentrant): pass
test_stats_arenas_summary (non-reentrant): pass
test_stats_arenas_small (non-reentrant): pass
test_stats_arenas_large (non-reentrant): pass
test_stats_arenas_bins (non-reentrant): pass
test_stats_arenas_lextents (non-reentrant): pass
--- pass: 7/7, skip: 0/7, fail: 0/7 ---
test_json_parser (non-reentrant): pass
test_json_parser (libc-reentrant): pass
test_json_parser (arena_new-reentrant): pass
test_stats_print_json (non-reentrant): pass
test_stats_print_json (libc-reentrant): pass
test_stats_print_json (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
unhooked_call (non-reentrant): pass
unhooked_call (libc-reentrant): pass
unhooked_call (arena_new-reentrant): pass
hooked_call (non-reentrant): pass
hooked_call (libc-reentrant): pass
hooked_call (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_ticker_tick (non-reentrant): pass
test_ticker_tick (libc-reentrant): pass
test_ticker_tick (arena_new-reentrant): pass
test_ticker_ticks (non-reentrant): pass
test_ticker_ticks (libc-reentrant): pass
test_ticker_ticks (arena_new-reentrant): pass
test_ticker_copy (non-reentrant): pass
test_ticker_copy (libc-reentrant): pass
test_ticker_copy (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
test_nstime_init (non-reentrant): pass
test_nstime_init (libc-reentrant): pass
test_nstime_init (arena_new-reentrant): pass
test_nstime_init2 (non-reentrant): pass
test_nstime_init2 (libc-reentrant): pass
test_nstime_init2 (arena_new-reentrant): pass
test_nstime_copy (non-reentrant): pass
test_nstime_copy (libc-reentrant): pass
test_nstime_copy (arena_new-reentrant): pass
test_nstime_compare (non-reentrant): pass
test_nstime_compare (libc-reentrant): pass
test_nstime_compare (arena_new-reentrant): pass
test_nstime_add (non-reentrant): pass
test_nstime_add (libc-reentrant): pass
test_nstime_add (arena_new-reentrant): pass
test_nstime_iadd (non-reentrant): pass
test_nstime_iadd (libc-reentrant): pass
test_nstime_iadd (arena_new-reentrant): pass
test_nstime_subtract (non-reentrant): pass
test_nstime_subtract (libc-reentrant): pass
test_nstime_subtract (arena_new-reentrant): pass
test_nstime_isubtract (non-reentrant): pass
test_nstime_isubtract (libc-reentrant): pass
test_nstime_isubtract (arena_new-reentrant): pass
test_nstime_imultiply (non-reentrant): pass
test_nstime_imultiply (libc-reentrant): pass
test_nstime_imultiply (arena_new-reentrant): pass
test_nstime_idivide (non-reentrant): pass
test_nstime_idivide (libc-reentrant): pass
test_nstime_idivide (arena_new-reentrant): pass
test_nstime_divide (non-reentrant): pass
test_nstime_divide (libc-reentrant): pass
test_nstime_divide (arena_new-reentrant): pass
test_nstime_monotonic (non-reentrant): pass
test_nstime_monotonic (libc-reentrant): pass
test_nstime_monotonic (arena_new-reentrant): pass
test_nstime_update (non-reentrant): pass
test_nstime_update (libc-reentrant): pass
test_nstime_update (arena_new-reentrant): pass
--- pass: 39/39, skip: 0/39, fail: 0/39 ---
test_tsd_main_thread (non-reentrant): pass
test_tsd_sub_thread (non-reentrant): pass
test_tsd_reincarnation (non-reentrant): pass
test_tsd_global_slow (non-reentrant): pass
--- pass: 4/4, skip: 0/4, fail: 0/4 ---
test_witness (non-reentrant): pass
test_witness (libc-reentrant): pass
test_witness (arena_new-reentrant): pass
test_witness_comp (non-reentrant): pass
test_witness_comp (libc-reentrant): pass
test_witness_comp (arena_new-reentrant): pass
test_witness_reversal (non-reentrant): pass
test_witness_reversal (libc-reentrant): pass
test_witness_reversal (arena_new-reentrant): pass
test_witness_recursive (non-reentrant): pass
test_witness_recursive (libc-reentrant): pass
test_witness_recursive (arena_new-reentrant): pass
test_witness_unlock_not_owned (non-reentrant): pass
test_witness_unlock_not_owned (libc-reentrant): pass
test_witness_unlock_not_owned (arena_new-reentrant): pass
test_witness_depth (non-reentrant): pass
test_witness_depth (libc-reentrant): pass
test_witness_depth (arena_new-reentrant): pass
--- pass: 18/18, skip: 0/18, fail: 0/18 ---
test_zero_small (non-reentrant): pass
test_zero_small (libc-reentrant): pass
test_zero_small (arena_new-reentrant): pass
test_zero_large (non-reentrant): pass
test_zero_large (libc-reentrant): pass
test_zero_large (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: lg_prof_sample:0
test_arena_reset (non-reentrant): pass
test_arena_reset (libc-reentrant): pass
test_arena_reset (arena_new-reentrant): pass
test_arena_destroy_initial (non-reentrant): pass
test_arena_destroy_initial (libc-reentrant): pass
test_arena_destroy_initial (arena_new-reentrant): pass
test_arena_destroy_hooks_default (non-reentrant): pass
test_arena_destroy_hooks_default (libc-reentrant): pass
test_arena_destroy_hooks_default (arena_new-reentrant): pass
test_arena_destroy_hooks_unmap (non-reentrant): pass
test_arena_destroy_hooks_unmap (libc-reentrant): pass
test_arena_destroy_hooks_unmap (arena_new-reentrant): pass
--- pass: 12/12, skip: 0/12, fail: 0/12 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: lg_prof_sample:0
test_alignment_errors (non-reentrant): pass
test_alignment_errors (libc-reentrant): pass
test_alignment_errors (arena_new-reentrant): pass
test_oom_errors (non-reentrant): pass
test_oom_errors (libc-reentrant): pass
test_oom_errors (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
test_zero_alloc (non-reentrant): pass
test_zero_alloc (libc-reentrant): pass
test_zero_alloc (arena_new-reentrant): pass
--- pass: 12/12, skip: 0/12, fail: 0/12 ---
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_extent_manual_hook (non-reentrant): pass
test_extent_manual_hook (libc-reentrant): pass
test_extent_manual_hook (arena_new-reentrant): pass
test_extent_auto_hook (non-reentrant): pass
test_extent_auto_hook (libc-reentrant): pass
test_extent_auto_hook (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_zero_alloc (non-reentrant): pass
test_zero_alloc (libc-reentrant): pass
test_zero_alloc (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
test_oom (non-reentrant): pass
test_oom (libc-reentrant): pass
test_oom (arena_new-reentrant): pass
test_remote_free (non-reentrant): pass
test_remote_free (libc-reentrant): pass
test_remote_free (arena_new-reentrant): pass
test_basic (non-reentrant): pass
test_basic (libc-reentrant): pass
test_basic (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_MALLOCX_ARENA (non-reentrant): pass
test_MALLOCX_ARENA (libc-reentrant): pass
test_MALLOCX_ARENA (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_alignment_errors (non-reentrant): pass
test_alignment_errors (libc-reentrant): pass
test_alignment_errors (arena_new-reentrant): pass
test_oom_errors (non-reentrant): pass
test_oom_errors (libc-reentrant): pass
test_oom_errors (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
test_grow_and_shrink (non-reentrant): pass
test_grow_and_shrink (libc-reentrant): pass
test_grow_and_shrink (arena_new-reentrant): pass
test_zero (non-reentrant): pass
test_zero (libc-reentrant): pass
test_zero (arena_new-reentrant): pass
test_align (non-reentrant): pass
test_align (libc-reentrant): pass
test_align (arena_new-reentrant): pass
test_lg_align_and_zero (non-reentrant): pass
test_lg_align_and_zero (libc-reentrant): pass
test_lg_align_and_zero (arena_new-reentrant): pass
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_basic (non-reentrant): pass
test_alignment_and_size (non-reentrant): pass
--- pass: 2/2, skip: 0/2, fail: 0/2 ---
<jemalloc>: Invalid conf pair: slab_sizes:1-4096:17|100-200:1|128-128:2
test_slab_sizes (non-reentrant): pass
test_slab_sizes (libc-reentrant): pass
test_slab_sizes (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_thread_arena (non-reentrant): pass
test_thread_arena (libc-reentrant): pass
test_thread_arena (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_same_size (non-reentrant): pass
test_same_size (libc-reentrant): pass
test_same_size (arena_new-reentrant): pass
test_extra_no_move (non-reentrant): pass
test_extra_no_move (libc-reentrant): pass
test_extra_no_move (arena_new-reentrant): pass
test_no_move_fail (non-reentrant): pass
test_no_move_fail (libc-reentrant): pass
test_no_move_fail (arena_new-reentrant): pass
test_size (non-reentrant): pass
test_size (libc-reentrant): pass
test_size (arena_new-reentrant): pass
test_size_extra_overflow (non-reentrant): pass
test_size_extra_overflow (libc-reentrant): pass
test_size_extra_overflow (arena_new-reentrant): pass
test_extra_small (non-reentrant): pass
test_extra_small (libc-reentrant): pass
test_extra_small (arena_new-reentrant): pass
test_extra_large (non-reentrant): pass
test_extra_large (libc-reentrant): pass
test_extra_large (arena_new-reentrant): pass
test_zero_large (non-reentrant): pass
test_zero_large (libc-reentrant): pass
test_zero_large (arena_new-reentrant): pass
--- pass: 24/24, skip: 0/24, fail: 0/24 ---
test_basic (non-reentrant): pass
test_basic (libc-reentrant): pass
test_basic (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_alignment_errors (non-reentrant): pass
test_alignment_errors (libc-reentrant): pass
test_alignment_errors (arena_new-reentrant): pass
test_oom_errors (non-reentrant): pass
test_oom_errors (libc-reentrant): pass
test_oom_errors (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
test_zero_alloc (non-reentrant): pass
test_zero_alloc (libc-reentrant): pass
test_zero_alloc (arena_new-reentrant): pass
--- pass: 12/12, skip: 0/12, fail: 0/12 ---
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_extent_manual_hook (non-reentrant): pass
test_extent_manual_hook (libc-reentrant): pass
test_extent_manual_hook (arena_new-reentrant): pass
test_extent_auto_hook (non-reentrant): pass
test_extent_auto_hook (libc-reentrant): pass
test_extent_auto_hook (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_zero_alloc (non-reentrant): pass
test_zero_alloc (libc-reentrant): pass
test_zero_alloc (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
test_oom (non-reentrant): pass
test_oom (libc-reentrant): pass
test_oom (arena_new-reentrant): pass
test_remote_free (non-reentrant): pass
test_remote_free (libc-reentrant): pass
test_remote_free (arena_new-reentrant): pass
test_basic (non-reentrant): pass
test_basic (libc-reentrant): pass
test_basic (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_MALLOCX_ARENA (non-reentrant): pass
test_MALLOCX_ARENA (libc-reentrant): pass
test_MALLOCX_ARENA (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_alignment_errors (non-reentrant): pass
test_alignment_errors (libc-reentrant): pass
test_alignment_errors (arena_new-reentrant): pass
test_oom_errors (non-reentrant): pass
test_oom_errors (libc-reentrant): pass
test_oom_errors (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
test_grow_and_shrink (non-reentrant): pass
test_grow_and_shrink (libc-reentrant): pass
test_grow_and_shrink (arena_new-reentrant): pass
test_zero (non-reentrant): pass
test_zero (libc-reentrant): pass
test_zero (arena_new-reentrant): pass
test_align (non-reentrant): pass
test_align (libc-reentrant): pass
test_align (arena_new-reentrant): pass
test_lg_align_and_zero (non-reentrant): pass
test_lg_align_and_zero (libc-reentrant): pass
test_lg_align_and_zero (arena_new-reentrant): pass
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_basic (non-reentrant): pass
test_alignment_and_size (non-reentrant): pass
--- pass: 2/2, skip: 0/2, fail: 0/2 ---
<jemalloc>: Invalid conf pair: slab_sizes:1-4096:17|100-200:1|128-128:2
test_slab_sizes (non-reentrant): pass
test_slab_sizes (libc-reentrant): pass
test_slab_sizes (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_thread_arena (non-reentrant): pass
test_thread_arena (libc-reentrant): pass
test_thread_arena (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_same_size (non-reentrant): pass
test_same_size (libc-reentrant): pass
test_same_size (arena_new-reentrant): pass
test_extra_no_move (non-reentrant): pass
test_extra_no_move (libc-reentrant): pass
test_extra_no_move (arena_new-reentrant): pass
test_no_move_fail (non-reentrant): pass
test_no_move_fail (libc-reentrant): pass
test_no_move_fail (arena_new-reentrant): pass
test_size (non-reentrant): pass
test_size (libc-reentrant): pass
test_size (arena_new-reentrant): pass
test_size_extra_overflow (non-reentrant): pass
test_size_extra_overflow (libc-reentrant): pass
test_size_extra_overflow (arena_new-reentrant): pass
test_extra_small (non-reentrant): pass
test_extra_small (libc-reentrant): pass
test_extra_small (arena_new-reentrant): pass
test_extra_large (non-reentrant): pass
test_extra_large (libc-reentrant): pass
test_extra_large (arena_new-reentrant): pass
test_zero_large (non-reentrant): pass
test_zero_large (libc-reentrant): pass
test_zero_large (arena_new-reentrant): pass
--- pass: 24/24, skip: 0/24, fail: 0/24 ---
test_basic (non-reentrant): pass
test_basic (libc-reentrant): pass
test_basic (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_alignment_errors (non-reentrant): pass
test_alignment_errors (libc-reentrant): pass
test_alignment_errors (arena_new-reentrant): pass
test_oom_errors (non-reentrant): pass
test_oom_errors (libc-reentrant): pass
test_oom_errors (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
test_zero_alloc (non-reentrant): pass
test_zero_alloc (libc-reentrant): pass
test_zero_alloc (arena_new-reentrant): pass
--- pass: 12/12, skip: 0/12, fail: 0/12 ---
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_extent_manual_hook (non-reentrant): pass
test_extent_manual_hook (libc-reentrant): pass
test_extent_manual_hook (arena_new-reentrant): pass
test_extent_auto_hook (non-reentrant): pass
test_extent_auto_hook (libc-reentrant): pass
test_extent_auto_hook (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
test_zero_alloc (non-reentrant): pass
test_zero_alloc (libc-reentrant): pass
test_zero_alloc (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
test_oom (non-reentrant): pass
test_oom (libc-reentrant): pass
test_oom (arena_new-reentrant): pass
test_remote_free (non-reentrant): pass
test_remote_free (libc-reentrant): pass
test_remote_free (arena_new-reentrant): pass
test_basic (non-reentrant): pass
test_basic (libc-reentrant): pass
test_basic (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_MALLOCX_ARENA (non-reentrant): pass
test_MALLOCX_ARENA (libc-reentrant): pass
test_MALLOCX_ARENA (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_alignment_errors (non-reentrant): pass
test_alignment_errors (libc-reentrant): pass
test_alignment_errors (arena_new-reentrant): pass
test_oom_errors (non-reentrant): pass
test_oom_errors (libc-reentrant): pass
test_oom_errors (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
test_grow_and_shrink (non-reentrant): pass
test_grow_and_shrink (libc-reentrant): pass
test_grow_and_shrink (arena_new-reentrant): pass
test_zero (non-reentrant): pass
test_zero (libc-reentrant): pass
test_zero (arena_new-reentrant): pass
test_align (non-reentrant): pass
test_align (libc-reentrant): pass
test_align (arena_new-reentrant): pass
test_lg_align_and_zero (non-reentrant): pass
test_lg_align_and_zero (libc-reentrant): pass
test_lg_align_and_zero (arena_new-reentrant): pass
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_basic (non-reentrant): pass
test_alignment_and_size (non-reentrant): pass
--- pass: 2/2, skip: 0/2, fail: 0/2 ---
<jemalloc>: Invalid conf pair: slab_sizes:1-4096:17|100-200:1|128-128:2
test_slab_sizes (non-reentrant): pass
test_slab_sizes (libc-reentrant): pass
test_slab_sizes (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_thread_arena (non-reentrant): pass
test_thread_arena (libc-reentrant): pass
test_thread_arena (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
test_same_size (non-reentrant): pass
test_same_size (libc-reentrant): pass
test_same_size (arena_new-reentrant): pass
test_extra_no_move (non-reentrant): pass
test_extra_no_move (libc-reentrant): pass
test_extra_no_move (arena_new-reentrant): pass
test_no_move_fail (non-reentrant): pass
test_no_move_fail (libc-reentrant): pass
test_no_move_fail (arena_new-reentrant): pass
test_size (non-reentrant): pass
test_size (libc-reentrant): pass
test_size (arena_new-reentrant): pass
test_size_extra_overflow (non-reentrant): pass
test_size_extra_overflow (libc-reentrant): pass
test_size_extra_overflow (arena_new-reentrant): pass
test_extra_small (non-reentrant): pass
test_extra_small (libc-reentrant): pass
test_extra_small (arena_new-reentrant): pass
test_extra_large (non-reentrant): pass
test_extra_large (libc-reentrant): pass
test_extra_large (arena_new-reentrant): pass
test_zero_large (non-reentrant): pass
test_zero_large (libc-reentrant): pass
test_zero_large (arena_new-reentrant): pass
--- pass: 24/24, skip: 0/24, fail: 0/24 ---
test_basic (non-reentrant): pass
test_basic (libc-reentrant): pass
test_basic (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof:true
test_alignment_errors (non-reentrant): pass
test_alignment_errors (libc-reentrant): pass
test_alignment_errors (arena_new-reentrant): pass
test_oom_errors (non-reentrant): pass
test_oom_errors (libc-reentrant): pass
test_oom_errors (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
test_zero_alloc (non-reentrant): pass
test_zero_alloc (libc-reentrant): pass
test_zero_alloc (arena_new-reentrant): pass
--- pass: 12/12, skip: 0/12, fail: 0/12 ---
<jemalloc>: Invalid conf pair: prof:true
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
<jemalloc>: Invalid conf pair: prof:true
test_extent_manual_hook (non-reentrant): pass
test_extent_manual_hook (libc-reentrant): pass
test_extent_manual_hook (arena_new-reentrant): pass
test_extent_auto_hook (non-reentrant): pass
test_extent_auto_hook (libc-reentrant): pass
test_extent_auto_hook (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
<jemalloc>: Invalid conf pair: prof:true
test_zero_alloc (non-reentrant): pass
test_zero_alloc (libc-reentrant): pass
test_zero_alloc (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
test_oom (non-reentrant): pass
test_oom (libc-reentrant): pass
test_oom (arena_new-reentrant): pass
test_remote_free (non-reentrant): pass
test_remote_free (libc-reentrant): pass
test_remote_free (arena_new-reentrant): pass
test_basic (non-reentrant): pass
test_basic (libc-reentrant): pass
test_basic (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
<jemalloc>: Invalid conf pair: prof:true
test_MALLOCX_ARENA (non-reentrant): pass
test_MALLOCX_ARENA (libc-reentrant): pass
test_MALLOCX_ARENA (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
test_alignment_errors (non-reentrant): pass
test_alignment_errors (libc-reentrant): pass
test_alignment_errors (arena_new-reentrant): pass
test_oom_errors (non-reentrant): pass
test_oom_errors (libc-reentrant): pass
test_oom_errors (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
<jemalloc>: Invalid conf pair: prof:true
test_grow_and_shrink (non-reentrant): pass
test_grow_and_shrink (libc-reentrant): pass
test_grow_and_shrink (arena_new-reentrant): pass
test_zero (non-reentrant): pass
test_zero (libc-reentrant): pass
test_zero (arena_new-reentrant): pass
test_align (non-reentrant): pass
test_align (libc-reentrant): pass
test_align (arena_new-reentrant): pass
test_lg_align_and_zero (non-reentrant): pass
test_lg_align_and_zero (libc-reentrant): pass
test_lg_align_and_zero (arena_new-reentrant): pass
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
<jemalloc>: Invalid conf pair: prof:true
test_basic (non-reentrant): pass
test_alignment_and_size (non-reentrant): pass
--- pass: 2/2, skip: 0/2, fail: 0/2 ---
<jemalloc>: Invalid conf pair: slab_sizes:1-4096:17|100-200:1|128-128:2
<jemalloc>: Invalid conf pair: prof:true
test_slab_sizes (non-reentrant): pass
test_slab_sizes (libc-reentrant): pass
test_slab_sizes (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
test_thread_arena (non-reentrant): pass
test_thread_arena (libc-reentrant): pass
test_thread_arena (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
<jemalloc>: Invalid conf pair: prof:true
test_same_size (non-reentrant): pass
test_same_size (libc-reentrant): pass
test_same_size (arena_new-reentrant): pass
test_extra_no_move (non-reentrant): pass
test_extra_no_move (libc-reentrant): pass
test_extra_no_move (arena_new-reentrant): pass
test_no_move_fail (non-reentrant): pass
test_no_move_fail (libc-reentrant): pass
test_no_move_fail (arena_new-reentrant): pass
test_size (non-reentrant): pass
test_size (libc-reentrant): pass
test_size (arena_new-reentrant): pass
test_size_extra_overflow (non-reentrant): pass
test_size_extra_overflow (libc-reentrant): pass
test_size_extra_overflow (arena_new-reentrant): pass
test_extra_small (non-reentrant): pass
test_extra_small (libc-reentrant): pass
test_extra_small (arena_new-reentrant): pass
test_extra_large (non-reentrant): pass
test_extra_large (libc-reentrant): pass
test_extra_large (arena_new-reentrant): pass
test_zero_large (non-reentrant): pass
test_zero_large (libc-reentrant): pass
test_zero_large (arena_new-reentrant): pass
--- pass: 24/24, skip: 0/24, fail: 0/24 ---
<jemalloc>: Invalid conf pair: prof:true
test_basic (non-reentrant): pass
test_basic (libc-reentrant): pass
test_basic (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_alignment_errors (non-reentrant): pass
test_alignment_errors (libc-reentrant): pass
test_alignment_errors (arena_new-reentrant): pass
test_oom_errors (non-reentrant): pass
test_oom_errors (libc-reentrant): pass
test_oom_errors (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
test_zero_alloc (non-reentrant): pass
test_zero_alloc (libc-reentrant): pass
test_zero_alloc (arena_new-reentrant): pass
--- pass: 12/12, skip: 0/12, fail: 0/12 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_extent_manual_hook (non-reentrant): pass
test_extent_manual_hook (libc-reentrant): pass
test_extent_manual_hook (arena_new-reentrant): pass
test_extent_auto_hook (non-reentrant): pass
test_extent_auto_hook (libc-reentrant): pass
test_extent_auto_hook (arena_new-reentrant): pass
--- pass: 6/6, skip: 0/6, fail: 0/6 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_zero_alloc (non-reentrant): pass
test_zero_alloc (libc-reentrant): pass
test_zero_alloc (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
test_oom (non-reentrant): pass
test_oom (libc-reentrant): pass
test_oom (arena_new-reentrant): pass
test_remote_free (non-reentrant): pass
test_remote_free (libc-reentrant): pass
test_remote_free (arena_new-reentrant): pass
test_basic (non-reentrant): pass
test_basic (libc-reentrant): pass
test_basic (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_MALLOCX_ARENA (non-reentrant): pass
test_MALLOCX_ARENA (libc-reentrant): pass
test_MALLOCX_ARENA (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_alignment_errors (non-reentrant): pass
test_alignment_errors (libc-reentrant): pass
test_alignment_errors (arena_new-reentrant): pass
test_oom_errors (non-reentrant): pass
test_oom_errors (libc-reentrant): pass
test_oom_errors (arena_new-reentrant): pass
test_alignment_and_size (non-reentrant): pass
test_alignment_and_size (libc-reentrant): pass
test_alignment_and_size (arena_new-reentrant): pass
--- pass: 9/9, skip: 0/9, fail: 0/9 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_grow_and_shrink (non-reentrant): pass
test_grow_and_shrink (libc-reentrant): pass
test_grow_and_shrink (arena_new-reentrant): pass
test_zero (non-reentrant): pass
test_zero (libc-reentrant): pass
test_zero (arena_new-reentrant): pass
test_align (non-reentrant): pass
test_align (libc-reentrant): pass
test_align (arena_new-reentrant): pass
test_lg_align_and_zero (non-reentrant): pass
test_lg_align_and_zero (libc-reentrant): pass
test_lg_align_and_zero (arena_new-reentrant): pass
test_overflow (non-reentrant): pass
test_overflow (libc-reentrant): pass
test_overflow (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_basic (non-reentrant): pass
test_alignment_and_size (non-reentrant): pass
--- pass: 2/2, skip: 0/2, fail: 0/2 ---
<jemalloc>: Invalid conf pair: slab_sizes:1-4096:17|100-200:1|128-128:2
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_slab_sizes (non-reentrant): pass
test_slab_sizes (libc-reentrant): pass
test_slab_sizes (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_thread_arena (non-reentrant): pass
test_thread_arena (libc-reentrant): pass
test_thread_arena (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
test_subthread (non-reentrant): pass
test_subthread (libc-reentrant): pass
test_subthread (arena_new-reentrant): pass
test_main_thread (non-reentrant): pass
test_main_thread (libc-reentrant): pass
test_main_thread (arena_new-reentrant): pass
--- pass: 15/15, skip: 0/15, fail: 0/15 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_same_size (non-reentrant): pass
test_same_size (libc-reentrant): pass
test_same_size (arena_new-reentrant): pass
test_extra_no_move (non-reentrant): pass
test_extra_no_move (libc-reentrant): pass
test_extra_no_move (arena_new-reentrant): pass
test_no_move_fail (non-reentrant): pass
test_no_move_fail (libc-reentrant): pass
test_no_move_fail (arena_new-reentrant): pass
test_size (non-reentrant): pass
test_size (libc-reentrant): pass
test_size (arena_new-reentrant): pass
test_size_extra_overflow (non-reentrant): pass
test_size_extra_overflow (libc-reentrant): pass
test_size_extra_overflow (arena_new-reentrant): pass
test_extra_small (non-reentrant): pass
test_extra_small (libc-reentrant): pass
test_extra_small (arena_new-reentrant): pass
test_extra_large (non-reentrant): pass
test_extra_large (libc-reentrant): pass
test_extra_large (arena_new-reentrant): pass
test_zero_large (non-reentrant): pass
test_zero_large (libc-reentrant): pass
test_zero_large (arena_new-reentrant): pass
--- pass: 24/24, skip: 0/24, fail: 0/24 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
test_basic (non-reentrant): pass
test_basic (libc-reentrant): pass
test_basic (arena_new-reentrant): pass
--- pass: 3/3, skip: 0/3, fail: 0/3 ---
<jemalloc>: Invalid conf pair: prof:true
<jemalloc>: Invalid conf pair: prof_active:false
`